go-controller/pkg/ovn/endpoints.go

```diff
@@ -267,3 +297,12 @@ func hasHostEndpoints(endpointIPs []string) bool {
 	}
 	return false
 }
+
+// When idling or empty LB events are enabled, we want to ensure we receive these packets and not reject them.
+func svcNeedsIdling(annotations map[string]string) bool {
+	if !config.Kubernetes.OVNEmptyLbEvents {
+		return false
+	}
+	_, ok := annotations[OvnServiceIdledAt]
```

Comment:
isn't the idling annotation now on the endpoints? i dont see anyway around using the openshift annotation, unless we use  a similar one and search for the annotation that says "idled-at" or something.

Reply:
we should double check, but last time I checked the idea was to move it to the Service, so there is no more doubts about endpoints/endpointslices

Reply:
Uhm, I think I miss some context here...
What I get is:
- we have a different annotation on services in ocp?
- the annotation is not on services but on endpoints (how does it work for slices?)

Any pointeres of where I can check?

Reply:
Uhm, I think I miss some context here...
What I get is:
- we have a different annotation on services in ocp?
- the annotation is not on services but on endpoints (how does it work for slices?)

Any pointeres of where I can check?

Reply:
you can do this in KIND, with an oc client:
```
[trozet@trozet ~]$ cat ~/webserver_deployment.yaml 
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: webserver-deployment
  labels:
    pod-name: server
    app: nginx
    role: webserver
spec:
  replicas: 2
  selector:
    matchLabels:
      app: nginx
  template:
    metadata:
      labels:
        app: nginx
        role: webserver
    spec:
      containers:
        - name: webserver1
          image: docker.io/centos/tools:latest
          command:
            - /sbin/init
          ports:
            - containerPort: 80
              name: serve-80
              protocol: TCP
```
```
[trozet@trozet ~]$ cat ~/service.yaml 
apiVersion: v1
kind: Service
metadata:
  name: my-service
spec:
  selector:
    role: webserver
#   pod-name: server
  ports:
    - protocol: TCP
      port: 80
      targetPort: 80
#    - name: server-81
#      port: 81
#      protocol: TCP
#      targetPort: 81
  externalIPs:
    - 80.11.12.10
```

```
[trozet@trozet ~]$ kubectl get svc
NAME         TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE
kubernetes   ClusterIP   10.96.0.1       <none>        443/TCP   76m
my-service   ClusterIP   10.96.148.104   80.11.12.10   80/TCP    1s
[trozet@trozet ~]$ kubectl get ep
NAME         ENDPOINTS                     AGE
kubernetes   172.19.0.3:6443               76m
my-service   10.244.1.4:80,10.244.1.5:80   3s
[trozet@trozet ~]$ 
[trozet@trozet ~]$ 
[trozet@trozet ~]$ 
[trozet@trozet ~]$ 
[trozet@trozet ~]$ 
[trozet@trozet ~]$ oc idle my-service
The service "default/my-service" has been marked as idled 
The service will unidle Deployment "default/webserver-deployment" to 2 replicas once it receives traffic 
Deployment "default/webserver-deployment" has been idled 

[trozet@trozet ~]$ kubectl get svc my-service -o yaml | grep idled-at
    idling.alpha.openshift.io/idled-at: "2021-03-25T21:58:54Z"
          f:idling.alpha.openshift.io/idled-at: {}
[trozet@trozet ~]$ kubectl get ep my-service -o yaml | grep idled-at
    idling.alpha.openshift.io/idled-at: "2021-03-25T21:58:54Z"
          f:idling.alpha.openshift.io/idled-at: {}
          f:idling.alpha.openshift.io/idled-at: {}


```

It looks like it is on both the service and the endpoints. I would suggest we just check the service for now, and i would make the postfix of the annotation the same between upstream and downstream "idled-at" then grep for annotation /idled-at in this code so we dont have to do some downstream carry.